setup.py

```diff
@@ -15,7 +15,7 @@
       ],
       extras_require={
           'dev': [
-              'ipdb==0.11',
+              'ipdb',
```

Comment:
Any reason to remove specific ipdb version?

Reply:
It is not compatible with latest setuptools version mentioned [here](https://github.com/singer-io/tap-impact/blob/master/.circleci/config.yml#L13). so cci has been failing from long [example](https://app.circleci.com/pipelines/github/singer-io/tap-impact/1064/workflows/97304bab-f54e-4331-b14c-a5301ce6e2d1/jobs/1066)